components/brave_vpn/brave_vpn_service_desktop.h

```diff
@@ -56,13 +56,18 @@ class BraveVpnServiceDesktop
   void RemoveObserver(Observer* observer);
 
   bool is_connected() const { return state_ == ConnectionState::CONNECTED; }
+  void CheckPurchasedStatus();
+  bool is_purchased_user() const { return is_purchased_user_; }
```

Comment:
I think this is fine for now - but eventually, we may have other states (like an enum). For example, if someone has it and then cancels; or if we add a suspended state (for abuse or something like this).

Reply:
yes, maybe we need more detailed state. This is added for doing test easily :)